src/screens/ModelPage/ModelPage.js

```diff
@@ -42,19 +42,49 @@ const LastDatesToAct = ({ model, outOfBoundsDate }) => {
     return `${month} ${day}`;
   }
 
+  function suggestedIntervention() {
+    switch (intervention) {
+      case INTERVENTIONS.NO_ACTION:
+        return INTERVENTIONS.SHELTER_IN_PLACE;
+      case INTERVENTIONS.SOCIAL_DISTANCING:
+        return INTERVENTIONS.SHELTER_IN_PLACE;
+      case INTERVENTIONS.SHELTER_IN_PLACE:
+        // TODO: Are we okay recommending lockdown?
+        return INTERVENTIONS.LOCKDOWN;
+      default:
+        return 'Stricter intervention';
+    }
+  }
+
+  let actionText, actionDateRange;
   if (
     !model.dateOverwhelmed ||
     model.dateOverwhelmed - new Date() > ONE_HUNDRED_DAYS
-  )
-    return <b>Greater than 3 months</b>;
+  ) {
+    actionText = `${intervention} projected to successfully delay hospital overload by greater than 3 months.`;
+  } else {
+    actionText = `To prevent hospital overload, you must implement ${suggestedIntervention()} by:`;
```

Comment:
Is 'you' the right choice here? Mostly thinking most of the viewers don't have that personal authority. They can individually do so.  Perhaps "To prevent hospital overload, ${suggestedIntervention()} must be implemented by:"?

Reply:
Good point. Changed.